src/mono/wasi/wasi.proj

```diff
@@ -226,17 +226,17 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <_FilesToCopy Include="$(MSBuildThisFileDirectory)..\wasm\runtime\runtime.c"   DestinationFolder="$(NativeBinDir)src" />
-      <_FilesToCopy Include="$(MSBuildThisFileDirectory)..\wasm\runtime\pinvoke.c"   DestinationFolder="$(NativeBinDir)src" />
+      <_FilesToCopy Include="$(BrowserProjectRoot)runtime\runtime.c"                 DestinationFolder="$(NativeBinDir)src" />
+      <_FilesToCopy Include="$(BrowserProjectRoot)runtime\pinvoke.c"                 DestinationFolder="$(NativeBinDir)src" />
       <_FilesToCopy Include="$(MSBuildThisFileDirectory)runtime/driver.c"            DestinationFolder="$(NativeBinDir)src" />
       <_FilesToCopy Include="$(MSBuildThisFileDirectory)runtime/main.c"              DestinationFolder="$(NativeBinDir)src" />
       <_FilesToCopy Include="$(MSBuildThisFileDirectory)runtime/stubs.c"             DestinationFolder="$(NativeBinDir)src" />
       <_FilesToCopy Include="$(MSBuildThisFileDirectory)runtime/synthetic-pthread.c" DestinationFolder="$(NativeBinDir)src" />
 
-      <_FilesToCopy Include="$(MonoProjectRoot)wasi\mono-include\driver.h"           DestinationFolder="$(NativeBinDir)include\wasm" />
-      <_FilesToCopy Include="$(MonoProjectRoot)wasm\runtime\gc-common.h"             DestinationFolder="$(NativeBinDir)include\wasm" />
-      <_FilesToCopy Include="$(MSBuildThisFileDirectory)..\wasm\runtime\runtime.h"   DestinationFolder="$(NativeBinDir)include\wasm" />
-      <_FilesToCopy Include="$(MSBuildThisFileDirectory)..\wasm\runtime\pinvoke.h"   DestinationFolder="$(NativeBinDir)include\wasm" />
+      <_FilesToCopy Include="$(WasiProjectRoot)mono-include\driver.h"           DestinationFolder="$(NativeBinDir)include\wasm" />
+      <_FilesToCopy Include="$(BrowserProjectRoot)runtime\gc-common.h"               DestinationFolder="$(NativeBinDir)include\wasm" />
```

Comment:
Should we move these sources to common directory?

Reply:
Yeah, in follow up PRs we can do the necessary moving around.